src/openms/include/OpenMS/DATASTRUCTURES/FASTAContainer.h

```diff
@@ -348,6 +348,25 @@ class DecoyHelper
     bool is_prefix; //< on success, was it a prefix or suffix
   };
 
+  // common decoy strings in FASTA files
+  // note: decoy prefixes/suffices must be provided in lower case
+  static std::vector<std::string> getAffixes()
+  {
+    return { "decoy", "dec", "reverse", "rev", "reversed", "__id_decoy", "xxx", "shuffled", "shuffle", "pseudo", "random" };
+  }
+
+  // returns a regex string to find decoy prefixes
+  static std::string getPrefixRegex()
+  {
+    return std::string("^(") + ListUtils::concatenate<std::string>(getAffixes(), "_*|") + "_*)";
+  }
+
+  // returns regex string to find decoy suffixes
+  static std::string getSuffixRegex()
+  {
+    return std::string("(") + ListUtils::concatenate<std::string>(getAffixes(), "_*|") + "_*)$";
```

Comment:
I know this was just refactored from existing code, by in my mind, this should be 
`return std::string("(*_") + ListUtils::concatenate<std::string>(getAffixes(), "|*_") +")$";`

Is there a test for this?